File: Python/Easy/First Non-Repeating Character/solution_1.py
def firstNonRepeatingCharacter(string):

    for first in range(len(string)):
        found = False
        for second in range(len(string)):

            if first != second and string[first] == string[second]:
                found = True
                
        if not found:
            return first
    
    return -1

File: Go/Easy/First Non-Repeated Character/solution_1.go
package main

func FirstNonRepeatingCharacter(str string) int {

	for first := range str {

		found := false

		for second := range str {

			if (first != second) && (str[first] == str[second]) {
				found = true
			}
		}

		if !found {
			return first
		}

	}

	return -1
}
